github/issues.go

```diff
@@ -44,6 +44,7 @@ type Issue struct {
 	Repository       *Repository       `json:"repository,omitempty"`
 	Reactions        *Reactions        `json:"reactions,omitempty"`
 	Assignees        []*User           `json:"assignees,omitempty"`
+	ClosedBy         *User             `json:"closed_by,omitempty"`
```

Comment:
@shurcooL I mean..on my side, the generator is ignoring my change with adding `closed_by`...

And inside the `github-accessors` file, there's no getter that returns type of either `User` or `Repository`... It seems to me that the genearator is ignoring all user-defined types...?